nuspecs/Microsoft.AspNet.SignalR.ServiceBus3.nuspec

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Microsoft.AspNet.SignalR.ServiceBus3</id>
```

Comment:
We can talk about this again. It's messy any way we go. We didn't like the idea of revving the entire SignalR package set to 3 because of this, but maybe we should. We need to decide if we're going to update to latest Katana packages too, which when through their own major version bump.